t/utils/utils.go

```diff
@@ -170,7 +171,7 @@ func GetHeadHashFromNetworkID(id int) string {
 	case params.StatusChainNetworkID:
 		return "0xe9d8920a99dc66a9557a87d51f9d14a34ec50aae04298e0f142187427d3c832e"
 	}
-
+	log.Debug("invalid network id", "networkid", id)
```

Comment:
Maybe some specific log with error for mainnet. So if anyone misuses the API he sees a useful message, like "tests shouldn't be run on the main network".